src/containers/tutor-home-page/add-photo-step/AddPhotoStep.style.js

```
@@ -1,71 +1,176 @@
+        </label>
+
+        <Typography sx={{ mt: '10px' }} variant='body2'>
+          {t('becomeTutor.photo.maxFileSize')}
+        </Typography>
+
+        <Box sx={{ mt: 'auto' }}>{btnsBox}</Box>
+      </Box>
+
+      <Snackbar
+        open={openSnackbar}
+        autoHideDuration={5000}
```

ESLint

[error] Props should be sorted alphabetically

(react/jsx-sort-props)